src/templates/default/Dockerfile

```
@@ -1,18 +1,13 @@
+# Build arguments
+ARG BUILD_PATH
+ARG NODE_VERSION
+
 # Setup
-FROM node:14-alpine AS image
+FROM node:$NODE_VERSION AS image
```

I love it! Alleen je alias mag weg
```suggestion
FROM node:$NODE_VERSION
```